src/content/en/fundamentals/security/credential-management/index.md

```diff
@@ -32,21 +32,16 @@ and take a look at the
 ### Check Credential Management API browser support
 
 Before using the Credential Management API,
-first check the API is supported,
-and that the version of the API is compatible with your code.
+first check the API is supported.
```

Comment:
We should update this to talk about specifically testing for whether PasswordCredential and FederatedCredential types are supported rather that the API.

Reply:
@engedy  I basically agree with you, but if I do that seriously, it will be a major rewrite to the majority of these documents. I'd rather leave it small changes for now and work on it once we start working on WebAuthn integration guide.